<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none;
background-color: #fff;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>565</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="outwear">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>40</y>
       <width>529</width>
       <height>391</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="widget_outwear">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>529</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="underwear">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>40</y>
       <width>529</width>
       <height>391</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="widget_underwear">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>527</width>
        <height>389</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_outwear">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #fff;</string>
   </property>
   <property name="text">
    <string>Outwear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_underwear">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #fff;</string>
   </property>
   <property name="text">
    <string>UnderWear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_footwear">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #fff;</string>
   </property>
   <property name="text">
    <string>Footwear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_jackets">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: #fff;</string>
   </property>
   <property name="text">
    <string>Jackets</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
